### ISSUE TITLE
"Add a generic font family" rule is applied for @font-face declarations

### ISSUE THREAD

**Issue by evgeny-konstantinov** [racodond/sonar-css-plugin#78]
Hi guys.

I'm using 4.9 version of the css plugin. It shows "Add a generic font family at the end of the declaration." issue for code like this:

```
@font-face {
    font-family: 'MyFont';
    src: url('../fonts/myfont.woff2') format('woff2'),
        url('../fonts/myfont.woff') format('woff'),
        url('../fonts/myfont.ttf') format('truetype');
    font-weight: normal;
    font-style: normal;
}
```

While it is even shown in the issue description that the rule should not be applied in the case.

> Note
> Obviously, this check does not apply to font-family descriptors in @font-face at-rules. For instance, the following piece of code does not raise any issue:
> @font-face {
>       font-family: 'My font family';
>       src: url('http://xxx');
> }

Is it possible to fix this?

**Comment by racodond** [comment]
Hi @evgeny-konstantinov,

I cannot reproduce with the latest version (4.12). Could you please give it a try: https://github.com/racodond/sonar-css-plugin/releases/tag/4.12 ?

If you can reproduce with the latest version, please provide a file to reproduce? Do you face the issue on CSS, SCSS or Less files?

Thank you

David

**Comment by racodond** [comment]
I'm closing the issue. Feel free to reopen it if you reproduce the issue with the latest version of the CSS plugin.
